core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -581,6 +640,18 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
    */
   final def interruptible: ZIO[R, E, A] = interruptStatus(InterruptStatus.Interruptible)
 
+  /**
+   * Returns an effect that when interrupted returns immediately.
+   * However, the effect and its interruption occurs in a forked fiber
+   */
+  final def interruptibleFork: ZIO[R, E, A] =
```

Comment:
I'd make sure this one has a `ZIO.interruptibleFork` variant for symmetry with `ZIO.interruptible`.

Reply:
do you mean add a `ZIO.interruptibleForkMask` ?